src/applications/caregivers/components/PreSubmitInfo/SubmitLoadingIndicator.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+const SubmitLoadingIndicator = ({ submission }) => {
+  const [isLoading, setLoading] = useState(false);
+
+  const hasAttemptedSubmit = submission.hasAttemptedSubmit;
+  const isSubmitPending = submission.status === 'submitPending';
+
+  // set loading to true if user has attempted to submit and submit is pending
+  useEffect(
+    () => {
+      if (hasAttemptedSubmit && isSubmitPending) {
+        setLoading(true);
+      }
+
+      return () => {
+        setLoading(false);
+      };
+    },
+    [hasAttemptedSubmit, isSubmitPending],
+  );
+
+  return (
+    <>
+      {isLoading && (
+        <div className="loading-container">
+          <div aria-live="polite" className="vads-u-margin-y--4">
```

Comment:
@LinnJS , quick dumb question for you (since I'm awful at checking code)-- is the entire aria-live container being dynamically added on submission here, or just the content within it? 

Reason I ask is because the ARIA live region must be present and must be empty _before_ making an announcement-- so the best approach is to ensure the ARIA live region is already on the page when the page loads. The region needs to be on the page, and the accessibility API needs to be aware that it is there and that it is empty before any announcements will be sent to the screen reader.

Otherwise, if you insert the ARIA live region after page load, you need to use JavaScript to delay (typically for at least one second) before sending any text into it to make an announcement.

Reply:
Before this was being dynamically added but its not any more... so that delay you were looking for is technically not there anymore. With that being said do we still need this live region at all? If we do need it I can add it to a wrapper above so it will be on page mount and then the content will be added on submission.

Reply:
Talked to Josh on a call and decided we do need the `aria-region` due to the content on the screen changing regardless of the content remaining static.